### PR TITLE
Update routes to conform to EIP-3091

### PR DESCRIPTION
**Pull request by csillag**
See #530 for context.

Tokens are not handled, since we don't have that functionality yet.

### CODE REVIEW

**Comment by matevz** [comment]
> See #530 for context.
> 
> Tokens are not handled yet.

I can confirm account and tx lookup with Metamask now works. Good job!

**Review by buberdds — APPROVED** [review]

File: .changelog/524.cfg.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Update routes to conform to EIP-3091
```
Comment (tjanez) [inline .changelog/524.cfg.md:1]: Please, add a link to EIP-3091 standard like:
```
Update routes to conform to [EIP-3091]

[EIP-3091]: https://eips.ethereum.org/EIPS/eip-3091
```

Comment (tjanez) [reply .changelog/524.cfg.md:1]: And classify it as a breaking change

Comment (csillag) [reply .changelog/524.cfg.md]: Added link, reanamed to breaking.

**Pull request merged**